onnxruntime/core/providers/cpu/math/element_wise_ops.cc

```diff
@@ -1070,9 +1070,16 @@ Status Expand_8<T>::Compute(OpKernelContext* context) const {
 }
 
 #define REG_EXPAND_KERNEL(TYPE)                                                    \
-  ONNX_CPU_OPERATOR_TYPED_KERNEL(                                                  \
+  ONNX_CPU_OPERATOR_VERSIONED_TYPED_KERNEL(                                        \
       Expand,                                                                      \
       8,                                                                           \
+      12,                                                                          \
+      TYPE,                                                                        \
+      KernelDefBuilder().TypeConstraint("T", DataTypeImpl::GetTensorType<TYPE>()), \
+      Expand_8<TYPE>);                                                             \
+  ONNX_CPU_OPERATOR_TYPED_KERNEL(                                                  \
+      Expand,                                                                      \
+      13,                                                                          \
       TYPE,                                                                        \
       KernelDefBuilder().TypeConstraint("T", DataTypeImpl::GetTensorType<TYPE>()), \
       Expand_8<TYPE>);
```

Comment:
It's unclear to me, since opset 8~13 can share the same kernel. do we still need to declare them desperately...

Reply:
yes it is the limitation of our current kernel registry api. if you declare it with 8-13, then this kernel won't be serving for the future opset, like opset14.